app/assets/stylesheets/main.scss

```diff
@@ -1,13 +1,17 @@
 .produce-background {
-  background-image: url("https://cdn1.sph.harvard.edu/wp-content/uploads/sites/21/2018/07/fruitveg.jpeg");
-  height: 900px;
-  opacity: 0.7;
+  background-image: linear-gradient(
+    rgba(255, 255, 255, 0.4),
+    rgba(255, 255, 255, 0.4)
+  ), url("https://cdn1.sph.harvard.edu/wp-content/uploads/sites/21/2018/07/fruitveg.jpeg");
```

Comment:
no rush, but consider hosting this image yourself on S3, or even something like Imgur. We dont want it to suddenly go down